os/fs/tmpfs/Kconfig

```diff
@@ -0,0 +1,73 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config FS_TMPFS
+	bool "TMPFS file system"
```

Comment:
Is TMPFS file system right? Isn't TMPFS TMP Files System?

Reply:
okay